Server/JokesOnYou.Web.Api/Repositories/Interfaces/IUserRepository.cs

```diff
@@ -12,6 +12,6 @@ public interface IUserRepository
         Task DeleteUserAsync(User user);
         Task<User> GetUserAsync(string id);
         Task<IEnumerable<User>> GetUsersAsync();
-        Task DeleteUserAsync(int id);
+        Task<bool> UpdateUser(User user);
```

Comment:
shouldnt use booleans, if something goes wrong we wanna throw an exception

Reply:
Yes, i removed the boolean return value